internal/toolexec/aspect/oncompile.go

```diff
@@ -125,19 +125,19 @@ func (w Weaver) OnCompile(cmd *proxy.CompileCommand) (err error) {
 	}
 
 	injector := injector.Injector{
-		Aspects:    aspects,
 		RootConfig: map[string]string{"httpmode": "wrap"},
 		Lookup:     imports.Lookup,
 		ImportPath: w.ImportPath,
 		TestMain:   cmd.TestMain() && strings.HasSuffix(w.ImportPath, ".test"),
+		ImportMap:  imports.PackageFile,
```

Comment:
I feel like in this context this should be inferred from the source file instead of manually configured.

Reply:
WDYM ?